App/Components/NotificationsBar.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Modal, Text, TouchableOpacity, Image } from 'react-native'
+import { Images } from '../Themes/'
+import StatusBarAlert from 'react-native-statusbar-alert'
+import PurpleGradient from './PurpleGradient'
+import styles from './Styles/ModalStyle'
+
+const NotificationScreen = ({notifications, onDismissModal}) => {
```

Comment:
Hi Robin, I think different companies have different practices on this. At my previous one, if we had a simple component that was only supporting the main one in a file, it was okay to just include it there rather than create a separate file. I'm ok with either preference.